arangod/Replication/ReplicationFeature.cpp

```diff
@@ -235,7 +235,8 @@ void ReplicationFeature::startApplier(TRI_vocbase_t* vocbase) {
   TRI_ASSERT(vocbase->type() == TRI_VOCBASE_TYPE_NORMAL);
   TRI_ASSERT(vocbase->replicationApplier() != nullptr);
 
-  if (vocbase->replicationApplier()->autoStart()) {
+  if (!ServerState::instance()->isClusterRole() &&
```

Comment:
war das kein noop vorher ?

Reply:
I think all this worked before already, because in cluster the calling function didn't call `startApplier`, but it was a little implicit and not obvious from looking at just this one function.